paper/paper.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'JankFlow: A Flexible Python Library for Organic Workflows'
+tags:
+  - molecular simulation
+  - materials science
+  - molecular dynamics
+  - polymers
+  - HOOMD-blue
+authors:
+  - name: Chris Jones
+    orcid: 0000-0002-6196-5274
+    equal-contrib: true
+    affiliation: 1
+  - name: Marjan Albooyeh
+    orcid: 0009-0001-9565-3076
+    equal-contrib: true
+    affiliation: 1
+  - name: Rainier Barrett
+    orcid: 0000-0002-5728-9074
+    equal-contrib: false
+    affiliation: 1
+  - name: Eric Jankowski
+    orcid: 0000-0002-3267-1410
+    corresponding: true
+    affiliation: 1
+affiliations:
+ - name: Boise State University, Boise, ID, USA
+   index: 1
+date: 01 January 2001
+bibliography: paper.bib
+---
+# Summary
+`JankFlow` is a package for reproducibly performing complex HOOMD-blue [@hoomd_2019]
+simulation workflows. It enables the programmatic specification of tasks including
+definition of molecular structures, forcefield definition and application and chaining
+together simulation stages (e.g., shrinking, equilibration, simulating a sequence
+of ensembles, tensile testing) through an extensible set of Python classes.
+The modular design supports a library of complex workflows for organic
+macrmolecular and polymer simulations. Tutorials are provided to demonstrate
+package features and flexibility.
+
+
+# Statement of need
+
+High-level programmatic specification of molecular simulation workflows are
+needed for two reasons: First: They provide the information necessary for a
+simulation study to be reproducible, and Second: They minimize the cognitive
+load of getting started with running experiments [?].
```

Comment:
@RainierBarrett Did you have a citation in mind for this one?

Reply:
I had been thinking on putting the TRUE paper ref here, but where it's explained now feels better. I removed it in the latest push.